aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SyncProcessor.java

```diff
@@ -161,11 +164,15 @@ private SyncTime filterOutOldSyncTimes(SyncTime lastSyncTime) throws DataStoreEx
     private <T extends Model> Single<Iterable<ModelWithMetadata<T>>> syncModel(
             Class<T> modelClass, SyncTime syncTime) {
         final Long lastSyncTimeAsLong = syncTime.exists() ? syncTime.toLong() : null;
-        return Single.create(emitter -> {
+        return Single.<Iterable<ModelWithMetadata<T>>>create(emitter -> {
             final Cancelable cancelable =
                 appSync.sync(modelClass, lastSyncTimeAsLong, metadataEmitter(emitter), emitter::onError);
             emitter.setDisposable(asDisposable(cancelable));
-        });
+        }).doOnSuccess(results ->
+            LOG.verbose("Successfully sync'd down cloud state for model type = " + modelClass.getSimpleName())
+        ).doOnError(failureToSync ->
```

Comment:
Same here - can we get some detailed logged or are they logged elsewhere?

Reply:
Updated to:
```
LOG.warn("blah blah", failureToSync);
```